Makefile

```diff
@@ -25,6 +25,13 @@ check:
 	@echo "*** Running pylint ***"
 	PYTHONPATH=$(PYTHONPATH):./src/ ./tests/pylint/runpylint.py
 
+test:
+	@echo "*** Running tests ***"
+	PYTHONPATH=$(PYTHONPATH):./src/ $(PYTHON) -m nose -v --with-coverage --cover-erase --cover-branches \
+                        --cover-package=pylorax --cover-package=bin --cover-package=sbin --cover-inclusive
+	coverage report -m
+	[ -f "/usr/bin/coveralls" ] && [ -n "$(COVERALLS_REPO_TOKEN)" ] && coveralls || echo
```

Comment:
Where does coveralls come from? I don't see it in RHEL7 or in EPEL7.
We need to make sure that all of the tests can be run from a RHEL7 system without adding a bunch of 3rd party binary blobs (I think we should be ok using EPEL stuff, eg. pylint, but should limit it to that)

Reply:
python-coveralls from EPEL

Reply:
Ok, I see python-coverage but not python-coveralls. I initially thought they were the same, but see they aren't so this looks ok.

Reply:
I see what's happening now. The python-coveralls package is built for EPEL but the last (only) available build has been removed, see:
https://koji.fedoraproject.org/koji/packageinfo?packageID=22032

Still I think if we want to use this service (I want) we can install one of the packages from Fedora or ask the EPEL maintainer to rebuild the package.

Reply:
Yeah, we could also help maintain it if we're going to depend on it.